iidm/iidm-extensions/src/main/java/com/powsybl/iidm/network/extensions/CoordinatedReactiveControlXmlSerializer.java

```diff
@@ -7,39 +7,25 @@
 package com.powsybl.iidm.network.extensions;
 
 import com.google.auto.service.AutoService;
+import com.powsybl.commons.extensions.AbstractExtensionXmlSerializer;
 import com.powsybl.commons.extensions.ExtensionXmlSerializer;
```

Comment:
Do you still need this import?

Reply:
Yes, it is called in the `@AutoService` annotation